corehq/apps/data_interfaces/utils.py

```diff
@@ -276,9 +276,17 @@ def run_rules_for_case(case, rules, now):
             ):
                 case = CaseAccessors(case.domain).get_case(case.case_id)
 
-        last_result = rule.run_rule(case, now)
-        aggregated_result.add_result(last_result)
-        if last_result.num_closes > 0:
-            break
+        try:
+            last_result = rule.run_rule(case, now)
+        except Exception:
+            notify_exception(None, "Error applying case update rule", {
```

Comment:
would an easy first step be to just log failure numbers in `CaseRuleActionResult` (and subsequently `DomainCaseRuleRun`)? Maybe that's out of scope for this PR. I'm also not sure how useful that model is, since clearly no one looks at it...

Reply:
Yeah, that would be nice, and maybe add an additional status like "finished, with errors".  I will say the model was useful for me to identify the problem, but it would be nice if it's displayed to the user - currently they just see the last time a rule was run (successfully), without any info on the runs themselves that I'm aware of.